shared/js/otSdkDriver.js

```diff
@@ -693,18 +694,7 @@ loop.OTSdkDriver = (function() {
         }
 
         channel.on({
-          message: function(ev) {
-            try {
-              var message = JSON.parse(ev.data);
-              /* Append the timestamp. This is the time that gets shown. */
-              message.receivedTimestamp = (new Date()).toISOString();
-
-              this.dispatcher.dispatch(
-                new sharedActions.ReceivedTextChatMessage(message));
-            } catch (ex) {
-              console.error("Failed to process incoming chat message", ex);
-            }
-          }.bind(this),
+          message: this._handleMessage.bind(this),
```

Comment:
Let's go ahead and handle the messages for a dedicated cursor channel, so we  can then leave this code for text chat messages untouched. The additional channel can be part of the 2nd commit.